code/modules/modular_computers/computers/item/pda.dm

```diff
@@ -253,23 +253,28 @@
 /obj/item/modular_computer/pda/nukeops
 	name = "nuclear pda"
 	device_theme = "syndicate"
-	has_pda_programs = FALSE
+	has_pda_programs = TRUE
 	comp_light_luminosity = 6.3 //matching a flashlight
 	light_color = COLOR_RED
 	greyscale_config = /datum/greyscale_config/tablet/stripe_thick
 	greyscale_colors = "#a80001#5C070F#000000"
-
+	long_ranged = TRUE
 	starting_programs = list(
-		/datum/computer_file/program/ntnetdownload/syndicate,
```

Comment:
I don't think the app is used anywhere else (and we don't want it used anywhere) so please delete it

Reply:
done